### PR TITLE
Added and tested for issue #15

### LINKED ISSUE #15: Allow the changing of the user-agent
We'd like to set the user-agent for Magellan-based requests to something like `RIPE Atlas Magellan`, so we need Cousteau to allow for this.

### PR DESCRIPTION
**Pull request by danielquinn**
It's a pretty simple change, but I took the opportunity to unify the styles in the __init__.  We had some cases of using `kwargs.get()` and others doing deliberate testing of `x in kwargs` -- as they're symantically the same, they're not literally the same.

Note that the `user_agent` bit there is using `default_user_agent` because it's possible that `user_agent=None` and we don't want to deal with that further down the pipe so we just force the default.

Closes #15

### CODE REVIEW

File: ripe/atlas/cousteau/request.py [hunk 1]
```diff
@@ -24,17 +24,14 @@ class AtlasRequest(object):
...
         self.url = ""
-        self.url_path = ""
-        self.key = kwargs.get("key", None)
-        if "url_path" in kwargs:
-            self.url_path = kwargs["url_path"]
-        if "server" in kwargs:
-            self.server = kwargs["server"]
-        else:
-            self.server = "atlas.ripe.net"
+        self.key = kwargs.get("key")
```
Comment (Jterbeest) [inline ripe/atlas/cousteau/request.py]: does this mean that the key is guaranteed to be there now? (as you dropped the None part...)

File: ripe/atlas/cousteau/request.py [hunk 2]
```diff
@@ -24,17 +24,14 @@ class AtlasRequest(object):
...
+        self.url_path = kwargs.get("url_path", "")
+        self.server = kwargs.get("server", "atlas.ripe.net")
```
Comment (Jterbeest) [inline ripe/atlas/cousteau/request.py]: although it's unlikely to change, I don't like to hard code this kind of infomation.

File: ripe/atlas/cousteau/request.py (hunk 1 above)
Comment (danielquinn) [inline ripe/atlas/cousteau/request.py]: None is the default, so it's not required.

File: ripe/atlas/cousteau/request.py (hunk 2 above)
Comment (danielquinn) [inline ripe/atlas/cousteau/request.py]: I'm not keen on hard-coding either, but I think when you're talking about the domain for the service, it's probably a safe bet for now.  Besides, not hard-coding would over-complicate the code for little benefit.

**Pull request closed without merging**